OrbitCore/CrashHandler.cpp

```diff
@@ -10,89 +10,45 @@
 #include "TcpClient.h"
 
 #ifdef _WIN32
-#include <client/windows/handler/exception_handler.h>
-
-#include <fstream>
-
-//-----------------------------------------------------------------------------
-void SendDumpInternal(const std::wstring& a_Dir, const std::wstring& a_Id);
-google_breakpad::ExceptionHandler* GHandler;
+#include <Shlwapi.h>
 
 //-----------------------------------------------------------------------------
-bool DmpFilter(void* context, EXCEPTION_POINTERS* exinfo,
-               MDRawAssertionInfo* assertion) {
-  return true;
+CrashHandler::CrashHandler(const std::wstring& dump_path) {
+  bool not_init = false;
+  if (is_init_.compare_exchange_strong(not_init, true)) {
+    Init(dump_path);
+  }
```

Comment:
This is great code! :-)

Just one thing: Is there an actual use case for having two CrashHandlers-objects in the same process?
If not I would say we don't want to work around a problem which is introduced by a developer. In this
case we want to fail hard at this point. We want to use the `CHECK` macro from `OrbitBase/Logging.h` to check
whether Init was called before and if it was, we want to fail hard here.

Reply:
I don't think that usage of two CrashHandlers in one program makes any sense.